TFNetworkRecLayer.py

```diff
@@ -6002,3 +6002,73 @@ def _get_output_native_rec_op(self, cell):
       y = y[0]
 
     return y
+
+
+class ZoneoutLSTMCell(BaseRNNCell):
+  """
+  Wrapper for tf LSTM to create Zoneout LSTM Cell.
+  This code is an adapted version of Rayhane Mamas version of Tacotron-2
```

Comment:
There should be a link here to the original code (Tacotron 2), and also to the paper.
(The formating is RST.)